src/app/ApplicationTemplate.Mobile/Android/MainActivity.Android.cs

```diff
@@ -7,7 +7,7 @@ namespace ApplicationTemplate;
 [Activity(
 	MainLauncher = true,
 	ConfigurationChanges = global::Uno.UI.ActivityHelper.AllConfigChanges,
-	WindowSoftInputMode = SoftInput.AdjustNothing | SoftInput.StateHidden,
+	WindowSoftInputMode = SoftInput.AdjustResize | SoftInput.StateHidden,
```

Comment:
The linked issue mentions `StateVisible` as well but I see it was not changed here.

Reply:
State visible should be used when you want the soft input to be displayed at navigation on the page.